app/shared/components/Wallet/Panel/Form/Stake/Confirming.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+import React, { Component } from 'react';
+import { Button, Header, Divider, Icon, Segment, Message } from 'semantic-ui-react';
+
+import StatsFetcher from '../../../../../utils/StatsFetcher';
```

Comment:
I'm very conflicted about the existence of this utility - this feels like something that should exist in state, not included in components to mutate values.

Reply:
yeah I wasn't sure about where to put it, should I create a helper folder under components and put it there?